com.unity.render-pipelines.core/Editor/CoreEditorDrawers.cs

```diff
@@ -158,7 +158,13 @@ void IDrawer.Draw(TData data, Editor owner)
                     m_ActionDrawers[i](data, owner);
 
                 if (m_Anim != null)
+                {
                     CoreEditorUtils.EndAdditionalPropertiesHighlight();
+
+                    // While the highlight is being changed, force the Repaint of the editor
+                    if (m_Anim.value > Mathf.Epsilon)
```

Comment:
is there any "is zero-ish" functionality in Maths or C#?

Reply:
I do not know, I've been looking at it until I found this solution.

Reply:
```
if (!Mathf.Approximately(m_Anim.value, 0f))
```
But basically it do the same with more cost. So I think this is ok (I dunno any other one)

Can the animbool can go other way too ?

Reply:
when the show starts it is put to 1 and it is going towards 0

Reply:
I think it is ok.

Just to be sure: Why not just `if (m_Anim.value > 0f)`? It seams like we update until we reach 0 right?

Reply:
Floating point imprecision makes comparing floats using the equals operator inaccurate.

Reply:
Yes you don't use equal here. So I don't get the issue. Beside sign is encoded separatly when you inspect float encoding so it cannot miss normally when checking >0f. Am I wrong ?

Reply:
yes your are right, I will change it